Cargo.toml

```diff
@@ -235,3 +235,6 @@ inherits = "release"
 lto = "fat"
 codegen-units = 1
 incremental = false
+
+[patch.crates-io]
+curve25519-dalek = { git = "https://github.com/jimmygchen/curve25519-dalek.git" }
```

Comment:
good practice to pin a `rev = "24019783e9bb9dc1464e7e503732f273a69969c6"` here so we're not vulnerable to supply chain attacks by sometime in control of the default branch on `jimmygchen/curve25519-dalek`

the Cargo.lock would only protect builds using `--locked`

Reply:
Good tip, thanks! 
Updated in https://github.com/jimmygchen/lighthouse/commit/e2f5d3af0034b984e36b4927ab866ccbbe14bbc6